conda-recipe/run_test.sh

```diff
@@ -0,0 +1 @@
+nosetests -v metadatastore --cover-package metadatastore --with-coverage
```

Comment:
Yes, i mean run the full test suite.

> Even on Travis where those tests were just run?

Yes. Those tests were run on the git repository, not the built conda package.  History has shown that it is quite simple for those two to *not* be equivalent.